docs/diginstroom/sip/2.0/sip_structure/5_structure_package.md

```diff
@@ -635,7 +635,7 @@ This means that the `dmdSec` MUST use `<mdRef>` elements to reference the extern
 | Attribute | `mets/dmdSec/mdRef/@xlink:href` |
 |-----------------------|-----------|
 | Name | Resource location |
-| Description | Indication of the actual location of the descriptive metadata file.<br>As indicated by the `@LOCTYPE` attribute, this filepath MUST be a URL type filepath.  |
+| Description | Indication of the actual location of the descriptive metadata file.<br>As indicated by the `@LOCTYPE` attribute, this filepath MUST be a URL type filepath. Furthermore, the filepath MUST be relative to the METS.xml file and MUST NOT use any dot notation.  |
```

Comment:
@milvld @spacid the prohibition of using dot notation is debatable because this is more a bug in the commons-ip rather than a spec requirement.

Reply:
I agree that if it's solely due to a bug in the validator code, we shouldn't enforce it. I wasn't sure whether it was an official E-ARK requirement. I don't think there is a requirement stating it, but all of their examples and samples don't use it, which I took as 'probably better not to do it with dot notation'.